src/MinimalDelegation.sol

```diff
@@ -188,4 +188,7 @@ contract MinimalDelegation is IERC7821, IKeyManagement, ERC1271, EIP712, ERC4337
             isValid = key.verify(_hash, signature);
         }
     }
+
+    /// @dev Allow the contract to receive ETH.
```

Comment:
Maybe inherit a receiver contract? Generally like that inheritance pattern even if you just put this into our own contract. Idt openzeppelin/contracts has a receiver contract but this is a nice util to have in general

Reply:
Ah good callout lets just use solady receiver